src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -57,7 +56,7 @@ public PersonCard(Person person, int displayedIndex) {
         this.person = person;
         id.setText(displayedIndex + ". ");
 
-        if (person.isDoctor()) {
+        if (person.getRole().toString().equals("Doctor")) {
```

Comment:
I think this is equivalent to person.isDoctor() (?) Would it be simpler to utilise that method?